### PR TITLE
Supplemental information

### PR DESCRIPTION
**Pull request by zietzm**
This builds a standalone PDF locally using `build.sh`. I'm not sure how to make this be built using CI here, or if we want that. I can push a PDF of the supplement if desired.

References https://github.com/greenelab/xswap-manuscript/issues/69

### CODE REVIEW

File: content/90.back-matter.md [hunk 1]
```diff
@@ -177,9 +87,7 @@ $$\frac{d(u_i)d(v_j)}{m - d(u_i) - d(v_j) + 1 + d(u_i)d(v_j)}.$$
 Unfortunately, we found that the above edge prior approximation is a poor approximation in many cases.
 We found that the following modified form (introduced in Methods) affords a superior approximation:
 
-\begin{equation}
-    P_{i,j} = \frac{d(u_i) d(v_j)}{\sqrt{(d(u_i) d(v_j))^2 + (m - d(u_i) - d(v_j) + 1)^2}}
-\end{equation}
+$$P_{i,j} = \frac{d(u_i) d(v_j)}{\sqrt{(d(u_i) d(v_j))^2 + (m - d(u_i) - d(v_j) + 1)^2}}$$
```
Comment (zietzm) [inline content/90.back-matter.md:90]: Not sure why, but the previous version rendered the equation in red on my machine. Going to `$$` fixes this.

**Comment by dhimmel** [comment]
> This build a standalone PDF locally using build.sh. I'm not sure how to make this be built using CI here

We should be able to get the desired effect with a workflow dispatch / adhoc CI job [here](https://github.com/greenelab/xswap-manuscript/actions/workflows/manubot.yaml). I'll configure like the following:

image

Running in this [job](https://github.com/greenelab/xswap-manuscript/actions/runs/7357539602)... will update comment with outputs uploaded as artifact.

[manuscript-7357539602-55ccf90.zip](https://github.com/greenelab/xswap-manuscript/files/13794343/manuscript-7357539602-55ccf90.zip)

Hmm, the build ended up building the `main` branch and not `supplement`. Let me look into why

**Comment by dhimmel** [comment]
> Let me look into why

Ah, that's because the supplement branch in the greenelab repo doesn't have the PR commit yet. I will merge this PR, and then we can make subsequent commits to the `supplement` branch in `greenelab/xswap-manuscript`

Post-merge running [job](https://github.com/greenelab/xswap-manuscript/actions/runs/7357607789): 
[manuscript-7357607789-303d5a5.zip](https://github.com/greenelab/xswap-manuscript/files/13794409/manuscript-7357607789-303d5a5.zip)

PDF looks good. I'll play around with 2 small updates:

- adding the article title somehow
- seeing if we can prefix figure and table numbers with `S`

**Comment by dhimmel** [comment]
Made some tweaks in https://github.com/greenelab/xswap-manuscript/commit/99b1ad9af7ba7814a2eaed76949abb7fc41b9266, ran a [dispatch](https://github.com/greenelab/xswap-manuscript/actions/runs/7357911735), and uploading [`manuscript-7357911735-99b1ad9.zip`](https://github.com/greenelab/xswap-manuscript/files/13794667/manuscript-7357911735-99b1ad9.zip). Extracted PDF: [`2023-12-29_xswap-supplementary-information.pdf`](https://github.com/greenelab/xswap-manuscript/files/13794676/2023-12-29_xswap-supplementary-information.pdf)

**Pull request merged**